src/category_theory/monoidal/category.lean

```diff
@@ -439,6 +439,25 @@ def tensor_right (X : C) : C ⥤ C :=
 
 variables (C)
 
+/--
+Tensoring on the left, as a functor from `C` into endofunctors of `C`.
+
+TODO: show this is a op-monoidal functor.
+-/
+@[simps]
+def tensoring_left : C ⥤ (C ⥤ C) :=
```

Comment:
```suggestion
def tensoring_left : C ⥤ C ⥤ C :=
```
Something similar can also be done with `tensoring_right`.

Reply:
Done.